src/OpenBinary/reader.jl

```diff
@@ -325,7 +346,11 @@ function PSRI.open(
     @assert last == position(ret.io)
 
     # go back to begning and initialize
-    seek(ret.io, 0)
+    if single_binary
```

Comment:
is this "if" needed?
ret.hs isnt already zero?
so `seek(ret.io, ret.hs)` works for both...